monitoring/harmony-monitor/blockchain-watchdog/reporting-server.go

```diff
@@ -931,7 +890,7 @@ func (m *monitor) statusSnapshot() statusReport {
 
 	usedSeats := 0
 	for _, info := range m.SuperCommittee.CurrentCommittee.Deciders {
-		usedSeats =+ linq.From(info.Committee).CountWith(func (v interface{}) bool {
+		usedSeats += linq.From(info.Committee).CountWith(func (v interface{}) bool {
```

Comment:
okay til. what's the diff?

Reply:
First one is the same as `usedSeats = +(linq...`
Seconds actually adds the `linq` value to `usedSeats`